### ISSUE TITLE
Change Streams/Chats more efficiently

### ISSUE THREAD

**Issue by Civrock1** [CBenni/mt2#50]
If feasible, I'd love to have an option to be able to change the stream or chat tabs to another person without having to close it, then having to find/pull the new one via the Home tab.

This may be something that's already in mind with the upcoming layout options or custom URLs, but another site that I think handles this very well is [multistre.am](https://multistre.am). Preset layouts and simple menu to add Twitch streamers you want to watch, then it opens them based on the amount of names entered. Available presets change accordingly. Alternatively, just change the usernames in the browser URL.

See some screenshots below. Don't mind the Light Mode chats, the 4/20 Twitch update broke Dark Mode and other things... (no idea if the multistre.am folks are still actively maintaining it).

![snag-0000](https://user-images.githubusercontent.com/19143373/39090394-d10599be-45ab-11e8-9db2-07a5af8f3f31.png)
![snag-0001](https://user-images.githubusercontent.com/19143373/39090395-d10f4cd4-45ab-11e8-91da-6160e71aed83.png)

**Comment by CBenni** [comment]
Yeah, something like it is planned. I will be working on a major layouts overhaul next week,
